oak_remote_attestation_verification/src/tests.rs

```diff
@@ -43,6 +43,9 @@ const LOG_ENTRY_PATH: &str = "testdata/logentry.json";
 // https://rekor.sigstore.dev/api/v1/log/publicKey.
 const REKOR_PUBLIC_KEY_PATH: &str = "testdata/rekor_public_key.pem";
 
+// Pretend the tests run at this time: 1 Nov 2023, 9:00 UTC
+const NOW_UTC_MILLIS: i64 = 1698829200000;
```

Comment:
(future) Would something like https://docs.rs/chrono/latest/chrono/ help make this more readable at some point?